elec_loc_labels.csv

```diff
@@ -1,4 +1,4 @@
-﻿subject_ID,label,long_name
+subject_ID,label,long_name
```

Comment:
Do you know what's going on here?

Reply:
As in, why that's in the changelog even though nothing changed?

Reply:
yeah

Reply:
Not a clue; I just opened it in excel, added the line, and closed. Maybe a windows vs mac file encoding thing like EOF's or temp's sometimes are?

Reply:
yeah, probably something like that